src/components/DocumentationTopic/ContentTableSection.vue

```diff
@@ -40,7 +41,8 @@ export default {
   props: {
     title: {
       type: String,
-      required: true,
+      default: '',
```

Comment:
Why `''` as a default value?

Reply:
I realize it still works with the conditional above, but it seems odd to me as opposed just making it optional without a default.